xapian-applications/omega/templates/query

```diff
@@ -108,8 +108,8 @@ $html{$date{$field{modtime},%Y-%m-%d}}</time></span>
 </noscript>
 }
 $if{$field{language},Language: <b>$html{$field{language}}</b>}
-$if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesize{$field{size}}}</b></span>}
-<div title="Mark this document as relevant" style="float:left;background-color:#cfc;border:1px solid green;margin-top:2px;">
+$if{$field{size},<br/><span title="$html{$field{size}} bytes">Size: <b>$html{$filesize{$field{size}}}</b></span>}
```

Comment:
You've introduced the `<br/>` here. Two things: under html5 it should just be `<br>`, and you also shouldn't need it (because you can do things like setting the subsequent element to `display: block;` to achieve the line break).

(We shouldn't attempt to make this "XHTML5" compatible, in my view. It's a pain, and not useful to many people.)